packages/clay-core/src/nav/Link.tsx

```diff
@@ -52,6 +52,40 @@ export const Link = React.forwardRef<
 		},
 		ref
 	) => {
+		if (showIcon) {
```

Comment:
This change here will probably break tests in DXP due to `button` being changed to `div`.